zapisy/templates/consent-dialog.html

```diff
@@ -5,9 +5,7 @@
         <div class="modal-content">
             <div class="modal-header">
                 <h5 class="modal-title" id="consentDialogLongTitle">Twoje dane osobowe w Systemie Zapisów</h5>
-                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
-                    <span aria-hidden="true">&times;</span>
-                </button>
+                <button type="button" class="btn-close" data-btn-dismiss="modal" aria-label="Close"></button>
```

Comment:
To też wygląda na typo, i ten przycisk też nie działa (a działał).
```suggestion
                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
```